docs/bulma-start.html

```diff
@@ -30,11 +30,16 @@
 
 {% include elements/anchor.html name="Install" %}
 
-{% highlight bash %}npm install bulma-start{% endhighlight %}
+<a href="https://github.com/jgthms/bulma-start/archive/master.zip" rel="nofollow noopener">Download bulma-start (.zip)</a>, rename to your project's name (i.e. `my-new-bulma-site`) and:
 
-<p class="block"><em>or</em></p>
+{% highlight bash %}
+cd my-new-bulma-site
+rm -rf .git        # remove original git settings
```

Comment:
I don't think this part is necessary anymore. The `.zip` doesn't include any `.git` folder.